measures/2019/measures-data.json

```diff
@@ -242,7 +266,7 @@
     "isHighPriority": false,
     "isInverse": false,
     "isIcdImpacted": false,
-    "isToppedOutByProgram": false,
+    "isToppedOutByProgram": true,
```

Comment:
@sarahnw just a heads up that a bunch of isToppedOutByProgram values are changing, presumably intended

Reply:
yep! we didn't have that info until we ran benchmarks so we have it now :)